build.sbt

```diff
@@ -8,5 +8,6 @@ libraryDependencies ++= Seq(
   "org.scorexfoundation" %% "scorex-core" % "2.0.0-M3-feature-handshake",
   "com.madgag.spongycastle" % "core" % "1.53.0.0",
   "org.scalatest" %% "scalatest" % "3.0.1" % "test",
-  "org.scalacheck" %% "scalacheck" % "1.13.4" % "test"
-)
+  "org.scalacheck" %% "scalacheck" % "1.13.4" % "test",
+  "org.scorexfoundation" %% "iodb" % "0.+"
```

Comment:
@adamsmo btw, this is how Scorex Example projects included it...maybe it's because they won't break the API but will release fixes....i can fix the version though